src/get-started/java-binding.md

```diff
@@ -795,6 +816,7 @@ An example of API service tests can be found in
 [junit-register-extension]: https://junit.org/junit5/docs/5.5.0/api/org/junit/jupiter/api/extension/RegisterExtension.html
 [junit-test]: https://junit.org/junit5/docs/5.5.0/api/org/junit/jupiter/api/Test.html
 [node-submit-transaction]: https://exonum.com/doc/api/java-binding/0.8.0/com/exonum/binding/core/service/Node.html#submitTransaction(com.exonum.binding.transaction.RawTransaction)
+[system-time-provider]: https://exonum.com/doc/api/java-binding/0.8.0/com/exonum/binding/testkit/TimeProvider.html#systemTime
```

Comment:
These links are not valid yet, but would be updated with the new release - is it okay to keep them like that?

Reply:
That's fine, we will replace the versions.